### DISCUSSION TITLE
Bitrate of "high" quality option

### DISCUSSION THREAD

**Discussion by max5354** [z-huang/music#406] [General]
What is the quality in bitrate of "high" quality song option.

**Comment by z-huang** [comment]
The highest bitrate stream we get from YT.

**Comment by JPGuillemin** [comment]
"High" is usually around 160kbps OGG-OPUS (VBR), which is CD quality (FLAC equivalent) with full 20KHz bandwidth.
OPUS codec reaches transparency at 128kbps with nearly no earable artifacts, so 160kbps OPUS is really overkill quality (theorically better than Spotify Premium).
Some songs are AAC 160kbps which is similar quality but limited to 17KHz bandwith.
